src/mscorlib/shared/System/Diagnostics/Tracing/EventSourceException.cs

```diff
@@ -18,9 +18,6 @@ namespace System.Diagnostics.Tracing
     /// <summary>
     /// Exception that is thrown when an error occurs during EventSource operation.
     /// </summary>
-#if !ES_BUILD_PCL
-    [Serializable]
```

Comment:
Thanks. I've added @vancem 's suggested ifdef along with !CORERT.

@brianrob , I've been testing this against CoreFX and modified tests appropriately so that it works. That change was https://github.com/dotnet/corefx/pull/20035.